spec/services/post_alerter_spec.rb

```diff
@@ -721,50 +721,47 @@ def set_topic_notification_level(user, topic, level_name)
   describe "push_notification" do
     let(:mention_post) { create_post_with_alerts(user: user, raw: 'Hello @eviltrout :heart:') }
     let(:topic) { mention_post.topic }
-
-    it "pushes nothing to suspended users" do
+    before do
       SiteSetting.allowed_user_api_push_urls = "https://site.com/push|https://site2.com/push"
-
-      evil_trout.update_columns(suspended_till: 1.year.from_now)
-
       2.times do |i|
         UserApiKey.create!(user_id: evil_trout.id,
                            client_id: "xxx#{i}",
                            application_name: "iPhone#{i}",
                            scopes: ['notifications'].map { |name| UserApiKeyScope.new(name: name) },
                            push_url: "https://site2.com/push")
       end
+    end
+
+    describe "DiscoursePluginRegistry#push_notification_filters" do
+      it "sends push notifications when all filters pass" do
+        Plugin::Instance.new.register_push_notification_filter do |user, payload|
```

Comment:
Just want to confirm that this doesn't leak the filter across tests and there is code somewhere the cleans this up after the test run?